src/gpl/src/nesterovPlace.cpp

```diff
@@ -479,8 +479,12 @@ int NesterovPlace::doNesterovPlace(int start_iter)
 
           log_->info(GPL,
                      107,
-                     "Timing-driven: RSZ delta area:     {}",
-                     block->dbuAreaToMicrons(nbc_->getDeltaArea()));
+                     "Timing-driven: repair_design (rsz) delta area:     "
```

Comment:
I think the `(rsz)` in this message is just noise. Just `"Timing-driven: repair_design delta area: "` should be enough.

Also, why the long whitespace here?

Reply:
no reason for the whitespace, I am gonna remove